README.md

```diff
@@ -1,14 +1,73 @@
+
+[sales@syncfusion.com](mailto:sales@syncfusion.com?Subject=Syncfusion Blazor Dropdown Menu - GitHUb) | [www.syncfusion.com](https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples) | Toll Free: 1-888-9 DOTNET
```

Comment:
This won't work
![image](https://user-images.githubusercontent.com/61148953/161230851-0f132db7-54bc-4d37-a450-c2c7d7f0c279.png)
use like this

![image](https://user-images.githubusercontent.com/61148953/161230893-dfaab52c-73b2-4ccb-be99-27a57e8756b6.png)